mappings/net/minecraft/client/render/debug/PathfindingDebugRenderer.mapping

```diff
@@ -1,10 +1,34 @@
 CLASS net/minecraft/class_868 net/minecraft/client/render/debug/PathfindingDebugRenderer
 	FIELD field_4615 pathTimes Ljava/util/Map;
 	FIELD field_4616 paths Ljava/util/Map;
+	FIELD field_4617 pathWeights Ljava/util/Map;
 	METHOD method_23117 drawPathLines (Lnet/minecraft/class_11;DDD)V
+		ARG 0 aiPath
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
 	METHOD method_23118 drawPath (Lnet/minecraft/class_11;FZZDDD)V
+		ARG 0 path
+		ARG 1 weight
+		ARG 2 showNodes
```

Comment:
No, it still draws the nodes even if this is false. This boolean makes it draw some extra nodes, which we don't know what they do.